test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -160,16 +162,33 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 		skipItIf( envVariables.VIEWPORT_NAME === 'mobile' )( 'Save draft', async function () {
 			await editorPage.saveDraft();
 		} );
+
+		it( 'Publish post', async function () {
+			publishedURL = await editorPage.publish();
+		} );
```

Comment:
Because we are no longer viewing the post as the publishing user, split this off into its own step and only do the publishing step.